shuffle/package.json

```diff
@@ -29,12 +29,14 @@
     "@types/react-dom": "18.2.1",
     "autoprefixer": "10.4.14",
     "axios": "^1.3.6",
+    "class-variance-authority": "^0.7.0",
     "clsx": "^1.2.1",
     "dayjs": "^1.11.7",
     "framer-motion": "^10.12.4",
     "js-tiktoken": "^1.0.7",
     "lodash": "^4.17.21",
     "logrocket": "^5.0.0",
+    "lucide-react": "^0.268.0",
```

Comment:
We currently use this: @heroicons/react

I am happy to change if lucide is better for performance / usability or has more / nicer icons.